### PR TITLE
Add links to the documentation in the examples

### PR DESCRIPTION
**Pull request by lloydtabb**

### CODE REVIEW

File: samples/duckdb/faa_duckdb/flights.malloy [hunk 1]
```diff
@@ -26,22 +27,28 @@ source: airports is airports_base + {
 
 source: flights is flights_base + {
   primary_key: id2
+
+  // rename some fiels as from their physical names
```
Comment (gmaden) [inline samples/duckdb/faa_duckdb/flights.malloy:31]: typo: fiels -> fields

File: samples/duckdb/faa_duckdb/flights_2.malloy [hunk 2]
```diff
@@ -1,14 +1,32 @@
-import "flights.malloy"
 
 // carrier analysis.
 //  show where carrires fly, what kinds of planes they use
+// 
+// over view of carriers
```
Comment (gmaden) [inline samples/duckdb/faa_duckdb/flights_2.malloy:5]: typo: over view -> overview

File: samples/duckdb/faa_duckdb/flights_2.malloy [hunk 3]
```diff
@@ -1,14 +1,32 @@
...
+//   What percentage the market does a carrier have?
```
Comment (gmaden) [inline samples/duckdb/faa_duckdb/flights_2.malloy:6]: nit: Missing "of" 
What percentage of the market does a carrier have?

Comment (lloydtabb) [reply samples/duckdb/faa_duckdb/flights_2.malloy:6]: thanks

**Pull request merged**